Makefile.common

```diff
@@ -184,10 +181,6 @@ else
 	yamllint .
 endif
 
-# For backward-compatibility.
-.PHONY: common-staticcheck
-common-staticcheck: lint
```

Comment:
Please keep this as well

Reply:
I'm all for it, but there should be a reason to keep those things right?